### DISCUSSION TITLE
Question about Reentrance contract

### DISCUSSION THREAD

**Discussion by danail-vasilev** [OpenZeppelin/ethernaut#704] [General]
I wonder what is the purpose of this line:
```javascript
      if(result) {
        _amount;
      }
```
https://github.com/OpenZeppelin/ethernaut/blob/master/contracts/contracts/levels/Reentrance.sol#L22

**Comment by justAWanderKid** [comment]
short answer is: does nothing. but why it exist?

When You Make a Low Level Call, the an boolean value gets returned, so this is just a way so you dont get warning about not checking the returned value. to understand what i mean, just remove that if statement.

Usually We do this to check if the low level call failed, if yes then revert:

```javascript
if (!result) { 
    revert("Low Level Call Failed");
}
```
